apps/meteor/client/components/SortList/SortModeList.tsx

```diff
@@ -30,6 +31,8 @@ function SortModeList(): ReactElement {
 					text={t('Name')}
 					input={<RadioButton pis='x24' name='sidebarSortby' onChange={setToAlphabetical} checked={sidebarSortBy === 'alphabetical'} />}
 				/>
+
+				<OmnichannelSortingDisclaimer />
```

Comment:
why not just render this IF you are a agent/manager and the omnichannel is enabled?